File: Python/CC_PRACTICE/INTEST.py
# https://www.codechef.com/problems/INTEST

n, k = map(int, input().split())
c = 0
while n > 0:
    j = int(input())
    if j % k == 0:
        c = c + 1
    n = n - 1
print(c)

# (c) 2021, Vikramaditya V. Vajire
# https://github.com/vikramaditya
# This source code is licensed under the MIT License

File: chapter_one/CC_PRACTICE/INTEST.c
// https://www.codechef.com/problems/INTEST

#include <stdio.h>

int main()
{
  int n;
  int k;
  int c;
  int j;

  scanf("%d %d", &n, &k);
  c = 0;
  for(n; n>0; n--){
    scanf("%d", &j);
    if (j%k == 0){
      c++;
    }
  }
  printf("%d", c);
  return 0;
}
